config/jest.config.js

```diff
@@ -29,13 +30,20 @@ module.exports = {
         ],
     ],
     testRunner: 'jest-circus/runner',
-    testRegex: 'test/.*\\.test\\.js',
+    testRegex: 'test/.*\\.test\\.ts',
     testEnvironment: 'node',
     testPathIgnorePatterns: [
         '/node_modules/',
     ],
     moduleFileExtensions: [
         'js',
+        'ts',
         'json',
     ],
+    // temporarily prevent type checking until finished setting up
+    globals: {
```

Comment:
this should be removed now that the types are finished

Reply:
removed